test/peach_test.exs

```diff
@@ -1,40 +1,215 @@
 defmodule PeachTest do
+  # import CSVLixir
   use ExUnit.Case
   doctest Peach
 
-  test "greets the world" do
-    assert Peach.hello() == :world
-  end
-
+  @tag timeout: 600_000
   test "normalise_text" do
-    assert Peach.normalise_text() == :to_do
+    test_data = [
+      ["foo", "foo"],
+      ["\n", "\n"],
+      ["🚼", "🚼"],
+      [[500], "Ǵ"],
+      [[600], "ɘ"]
```

Comment:
@bduvenhage do you think you could provide some test examples here for normalizing text with NFKC? Feel free to pop them in this comment and I can add them to the code.

Reply:
@nathanbegbie Please use: 
"＋－．～）｝"  ==>  "+-.~)}"
"１２３"  ==>  "123"
"ａｂｃＡＢＣ"  ==>  "abcABC"

More examples at https://towardsdatascience.com/difference-between-nfd-nfc-nfkd-and-nfkc-explained-with-python-code-e2631f96ae6c

Reply:
@bduvenhage can you add those examples yourself? Might be a good intro-to-elixir challenge :)